x-pack/plugin/watcher/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -1,2 +1,6 @@
 ALL-UNNAMED:
   - manage_threads
+  - files:
+    - relative_path: ".mime.types"
+      relative_to: "home"
```

Comment:
Note, this will cause tests to break until home-relative file permissions are supported